src/container.c

```diff
@@ -271,6 +272,67 @@ static int container_setup_modules(struct hyper_container *container)
 	return 0;
 }
 
+static int container_setup_mount_cgroups() {
+	// Expects the current dir to be the container rootfs, and should be called from
+	// container_setup_mount()
+
+	// TODO: should this act like docker and set cgroups RO unless container is
+	// privileged?
+
+	// Mount cgroupfs hierarchy for the container
+	if (mount("none", "./sys/fs/cgroup", "tmpfs", NULL, NULL) == -1) {
+		perror("mount ./sys/fs/cgroup failed");
+		return -1;
+	}
+
+	void *handle;
+	struct controller_data item;
+	int ret;
+	ret = cgroup_get_all_controller_begin(&handle, &item);
```

Comment:
We don't want to introduce more library dependency to hyperstart, chould you implement these necessary libcgroup api in hyperstart?